webrtc.html

```diff
@@ -8099,7 +8101,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 </li>
                 <li>
                   <p>If the <code><a data-for="RTCDataChannel">id</a></code>
-                  attribute is uninitialized (not set via the dictionary),
+                  attribute is <code>null</code> (due to no ID being
+                  passed into <code>createDataChannel</code>),
                   and the DTLS role of the SCTP transport has already been
                   negotiated, then initialize <code><a data-for=
                   "RTCDataChannel">id</a></code> to a value generated by the
```

Comment:
Looking at https://tools.ietf.org/html/draft-ietf-rtcweb-data-protocol-09, the id is an "SCTP stream identifier (0-65534)". This step will either cause id to be a number, or an exception will be thrown, null is not a possible outcome AFACIT.